pkg/state/state_ofed.go

```diff
@@ -647,13 +653,18 @@ func (s *stateOFED) setEnvFromClusterWideProxy(cr *mellanoxv1alpha1.NicClusterPo
 // mergeWithDefaultEnvs returns env variables provided in currentEnvs merged with default
 // env variables for MOFED container.
 func (s *stateOFED) mergeWithDefaultEnvs(currentEnvs []v1.EnvVar) []v1.EnvVar {
-	if envVarsWithGet(currentEnvs).Get("CREATE_IFNAMES_UDEV") != nil {
-		// already exists dont overwrite
-		return currentEnvs
-	}
+	envs := currentEnvs
 
 	// CREATE_IFNAMES_UDEV: should be set to true if not provided.
-	return append(currentEnvs, v1.EnvVar{Name: "CREATE_IFNAMES_UDEV", Value: "true"})
+	if envVarsWithGet(currentEnvs).Get(envVarCreateIfNamesUdev) == nil {
+		envs = append(envs, v1.EnvVar{Name: envVarCreateIfNamesUdev, Value: "true"})
+	}
+	// USE_DRIVERS_INVENTORY: should be set to true if not provided.
+	if envVarsWithGet(currentEnvs).Get(envVarUseDriversInventory) == nil {
```

Comment:
hmmm,
i thought we will add : NVIDIA_NIC_DRIVERS_INVENTORY_PATH here with value pointing to`/mnt/drivers-inventory`

if provided with empty value ("") then it will override the default value above and disable the feature (which we may want if things go south with this feature.

Reply:
I see now! :) Updated the code to use the `NVIDIA_NIC_DRIVERS_INVENTORY_PATH` env var directly